jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -753,6 +754,16 @@ function configure_jbpm_cache() {
         </cache-container>\n<cache-container name="server" aliases="singleton cluster" default-cache="default" module="org.wildfly.clustering.server">#g' ${CONFIG_FILE}
 }
 
+function configure_jbpm_cache_modules() {
+    sed -i 's#<system export="true">#<module name="org.infinispan" services="export"/><module name="org.jgroups"/>\n<system export="true">#g' ${JBOSS_HOME}/standalone/deployments/ROOT.war/WEB-INF/jboss-deployment-structure.xml
+
+    if [ -d "/opt/kie/dependencies/jbpm-clustering" ] ;then
+      for f in /opt/kie/dependencies/jbpm-clustering/kie-server-services-jbpm-cluster-*.jar; do
```

Comment:
is there more than one dependency in there?

Reply:
Currently just one but is to avoid an hardcoded name

Reply:
if there isn't more than one, so, why use a loop?
cp and ln commands accept wildcard as well.

Reply:
ok, I'll change